misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/command/BindgenCommand.java

```diff
@@ -87,6 +114,20 @@ public void execute() {
             bindingsGenerator.setDependentJars(dependencyList);
         }
 
+        String splitColonRegex = "\\s*:\\s*";
+        if (this.mavenDependency != null) {
+            String[] mvnDependency = this.mavenDependency.split(splitColonRegex);
+            if (mvnDependency.length != 3) {
+                outError.println("\nError in the maven dependency provided.\n");
+                outStream.println(BINDGEN_CMD);
+                outStream.println("\nUse 'ballerina bindgen --help' for more information on the command.");
+                return;
+            }
+            bindingsGenerator.setMvnGroupId(mvnDependency[0]);
+            bindingsGenerator.setMvnArtifactId(mvnDependency[1]);
+            bindingsGenerator.setMvnVersion(mvnDependency[2]);
+        }
+
```

Comment:
Shall we throw `UnsupportedOperationException` in `setParentCmdParser(...)` in line number 164?

Reply:
All the other commands seem to be following this norm. I think we can keep this as it is.